certchain/certmanager/augmentor.go

```diff
@@ -130,13 +145,18 @@ func (a *Augmentor) daemon(rcNext, orNext futureevent.Event) {
 				orNext, err = a.maintainOCSPResp()
 				if err != nil {
 					log.Printf("cannot update the OCSP response: %v", err)
+				} else {
+					log.Printf("successfully updated OCSP response")
 				}
 			}
 		case <-orNext.Chan():
 			orNext, err = a.maintainOCSPResp()
 			if err != nil {
 				log.Printf("cannot update the OCSP response: %v", err)
+			} else {
+				log.Printf("successfully updated OCSP response")
 			}
+
```

Comment:
If you find this empty line increase the readability, add another above the line `case <-orNext.Chan()` for consistency. Otherwise, remove this empty line.

Reply:
Removed.